crates/spirv-builder/src/test/basic.rs

```diff
@@ -463,6 +463,9 @@ pub fn main() {
     let vector = glam::Vec2::new(1.0, 2.0);
     let element = unsafe { spirv_std::arch::vector_extract_dynamic(vector, 1) };
     assert!(2.0 == element);
+    let uvector = glam::UVec2::new(1, 2);
+    let element: u32 = unsafe { spirv_std::arch::vector_extract_dynamic(uvector, 1) };
```

Comment:
sorry, meant to include this in my comment too but didn't want to spam too many comments
```suggestion
    let element = unsafe { spirv_std::arch::vector_extract_dynamic(uvector, 1) };
```

Reply:
Spam me with review comments 😄